webv2/helpers/helpers.go

```diff
@@ -136,10 +148,9 @@ func CheckOrCreateMetadataDb(projectId string, instanceId string) (isExist bool,
 	}
 	if dbExists {
 		isExist = true
-		return
 	}
 
-	err = createDatabase(ctx, uri)
+	err = createDatabase(ctx, uri, isExist)
```

Comment:
Can we pass dbExists only instead for isExist?

Reply:
Done